### DISCUSSION TITLE
Android App Series Book Sorting

### DISCUSSION THREAD

**Discussion by troycarpenter** [advplyr/audiobookshelf-app#443] [Q&A] [answered]
I must be missing something, but I can't figure out how the books in a series are being presented.  When I select a series, the books are not presented in series order, but they are not presented in publication or title order either.
![image](https://user-images.githubusercontent.com/8419000/202569976-fc635dee-3b99-400c-9434-94552b5d49eb.png)
In this screenshot of the android app, you can see the books are not presented in series order (indicated by the number in the upper right corner of the book cover).

In the web interface, the series books are presented in the correct order:
![image](https://user-images.githubusercontent.com/8419000/202569540-92059d7d-7c87-4526-9e6a-1b06f5bfc6c4.png)
In this screenshot from the web GUI, you can see the books ARE in the series order.

Am I just missing something?

**Comment by advplyr** [comment, marked as answer]
You are not, this will be fixed in the next server release today or tomorrow.
